e2e/single-cluster/gitrepo_test.go

```diff
@@ -27,13 +27,15 @@ const (
 
 var _ = Describe("Monitoring Git repos via HTTP for change", Label("infra-setup"), func() {
 	var (
-		tmpdir           string
+		tmpDir           string
 		clonedir         string
 		k                kubectl.Command
 		gh               *githelper.Git
 		clone            *git.Repository
 		repoName         string
 		inClusterRepoURL string
+		gitrepoName      string
+		r                = rand.New(rand.NewSource(1))
```

Comment:
We could use `GinkgoRandomSeed()`
```suggestion
		r                = rand.New(rand.NewSource(GinkgoRandomSeed()))
```